.claude/commands/gen-release-notes.md

```
@@ -167,17 +167,69 @@ Generate a `CHANGELOG.md` entry using this exact format:
+Update the version in all package files to keep them in sync:
+
+### 6a. Python package (pyproject.toml)
+
+Update the version field in `pyproject.toml`:
+
+```bash
+# pyproject.toml - line 3
+sed -i '' 's/^version = ".*"/version = "X.Y.Z"/' pyproject.toml
+```
+
+### 6b. Python module (amelia/__init__.py)
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)